packages/core/src/util/file-type.ts

```diff
@@ -42,6 +42,12 @@ const XML_MAGIC_NUMBER_3 = 0x78;
 const XML_MAGIC_NUMBER_4 = 0x6d;
 const XML_MAGIC_NUMBER_5 = 0x6c;
 const XML_MAGIC_NUMBER_6 = 0x20;
+const XML_MAGIC_NUMBER_7 = 0x3c;
+const XML_MAGIC_NUMBER_8 = 0x43;
+const XML_MAGIC_NUMBER_9 = 0x6c;
+const XML_MAGIC_NUMBER_10 = 0x69;
+const XML_MAGIC_NUMBER_11 = 0x6e;
+const XML_MAGIC_NUMBER_12 = 0x69;
 const PNG_MAGIC_NUMBER_1 = 0x89;
 const PNG_MAGIC_NUMBER_2 = 0x50;
```

Comment:
what's the logic behind adding this? seems unrelated to v2

Reply:
Its just adding this edge case logic we have in the current IHEGateway code: https://github.com/metriport/metriport/blob/33dc2739a380cad94414fa3792e1e6e81b100fc2/packages/ihe-gateway/server/CodeTemplates/Common%20Library/File%20type/File%20type.js#L42

Basically some xmls start with <clinical document> as opposed to <?xml so its just accounting for that case. There are two different test xml files to reflect these two different cases